URA-695/normalise_vcf.sh

```
@@ -0,0 +1,10 @@
+## normalise the VCFs to deal with multiallelic variants
```

Shellcheck (0.10.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)